src/test/java/bio/terra/integration/DataRepoFixtures.java

```diff
@@ -629,6 +631,32 @@ public BulkLoadArrayResultModel bulkLoadArray(
     return null; // Make findbugs happy
   }
 
+  public BulkLoadResultModel bulkLoad(
+      TestConfiguration.User user, UUID datasetId, BulkLoadRequestModel requestModel)
+      throws Exception {
+
+    String json = TestUtils.mapToJson(requestModel);
+
+    DataRepoResponse<JobModel> launchResponse =
+        dataRepoClient.post(
+            user, "/api/repository/v1/datasets/" + datasetId + "/files/bulk", json, JobModel.class);
+    assertTrue("bulkLoad launch succeeded", launchResponse.getStatusCode().is2xxSuccessful());
+    assertTrue(
+        "bulkload launch response is present", launchResponse.getResponseObject().isPresent());
+
+    DataRepoResponse<BulkLoadResultModel> response =
+        dataRepoClient.waitForResponse(user, launchResponse, BulkLoadResultModel.class);
+    if (response.getStatusCode().is2xxSuccessful()) {
+      assertThat("bulkLoad is successful", response.getStatusCode(), equalTo(HttpStatus.OK));
+      assertTrue("ingestFile response is present", response.getResponseObject().isPresent());
+      return response.getResponseObject().get();
+    }
+    ErrorModel errorModel = response.getErrorObject().orElse(null);
+    logger.error("bulkLoad failed: " + errorModel);
+    fail();
+    return null; // Make findbugs happy
```

Comment:
It looks like this duplicates code in `getLoadHistory()` and `bulkLoadArray()`. If so, it would be better to find a way to share the code instead of copying it.

Also you can avoid the need to `return null;` if you replace `fail()` with `throw new AssertionError();`.

Reply:
I pulled some of the low-hanging fruit into a helper method. I feel like the rest is pretty reasonable to leave - calls w/ unique class or response references.